### PR TITLE
Added king class and movements

### PR DESCRIPTION
**Pull request by edgar-17**
"Added king class and movements in king.py"
"Added some king performance tests in main.py"
"Added changes needed in board.py"

### CODE REVIEW

File: board.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from king import King
```
Comment (edgar-17) [inline board.py:1]: adding king class

File: board.py [hunk 2]
```diff
@@ -13,7 +14,7 @@ def __init__(self) -> None:
             "        ",
             "        ",
             "pppppppp",
-            "        ",
+            "   k    ",
```
Comment (edgar-17) [inline board.py:17]: adding black king in initial board

**Review by PEnriqueGG — APPROVED** [review]
Great job! Approved LGTM

**Review by romeo-sayeg — APPROVED** [review]
nice job!!!, it fine to me.

**Pull request merged**